src/css/widget.scss

```diff
@@ -78,6 +78,7 @@
 	margin-bottom: -11px;
 }
 .thumbnail_explination {
+	display: table;

```

Comment:
If it's possible to do this using `inline-block` (or possibly `flex`), that would probably be preferable. Otherwise this is fine.

Reply:
the image is set to be a background on a `p` element. would have to change Materia PHP to fix properly. this should be fine for now (imo) if we're considering a rework of the catalog in the near future anyway

Reply:
Yeah, that makes sense. Would've been a stretch goal if anything.